usr/lib/user-manager/hooks.d/e17-wizard/after/85-special-languages-configurations.sh

```diff
@@ -102,7 +102,7 @@ main(){
             zenity --entry --text="$( eval_gettext "Switch to the new keyboard layout and type in any text here." )" || true
 
             # final note
-            zenity --info --text="$( eval_gettext "You can install multiple Ibus packages to support your language. If you find that there is a better alternative, report it to Elive, so it can be implemented and work for everybody." )" || true
+            zenity --info --text="$( eval_gettext "You can install multiple Ibus packages to support your language. If you find that there is a better alternative, report it to Elive, so it can be implemented and work for everyone else." )" || true
```

Comment:
I prefer "everybody".

Reply:
I say we do the actual proof reading and discussion in 'eltrans', not here.

Reply:
Yeah, but that was before we had special access to eltrans.